test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9443,7 +9443,7 @@ def make_input(batch_size):
                 h0 = torch.randn(directions * layers, batch_size, RNN_HIDDEN_SIZE)
                 inputs.append(h0)
             if packed_sequence != 0:
-                inputs.append(torch.IntTensor(seq_lengths))
+                inputs.append(torch.LongTensor(seq_lengths))
```

Comment:
Why is LongTensor of seq_lengths is preferred to IntTensor? Could you add a comment?

Reply:
This change is actually not needed - in first iteration of my fix, I moved whole as_tensor() in utils/rnn.py under "if not isTensor", so it stopped converting to int64 when tensor was passed. Then I corrected that.

Reply:
You only reverted one of them. Would you do all of them?